external/storm-kafka/src/jvm/storm/kafka/ZkState.java

```diff
@@ -33,20 +32,29 @@
 import java.util.Map;
 
 public class ZkState {
+
+    public static final String TRANSACTIONAL_ZOOKEEPER_SERVERS = "transactional.zookeeper.servers";
+    public static final String TRANSACTIONAL_ZOOKEEPER_PORT = "transactional.zookeeper.port";
+    public static final String STORM_ZOOKEEPER_SESSION_TIMEOUT = "storm.zookeeper.session.timeout";
+    public static final String STORM_ZOOKEEPER_CONNECTION_TIMEOUT = "storm.zookeeper.connection.timeout";
+    public static final String STORM_ZOOKEEPER_RETRY_TIMES = "storm.zookeeper.retry.times";
+    public static final String STORM_ZOOKEEPER_RETRY_INTERVAL = "storm.zookeeper.retry.interval";
+
+
     public static final Logger LOG = LoggerFactory.getLogger(ZkState.class);
     CuratorFramework _curator;
 
     private CuratorFramework newCurator(Map stateConf) throws Exception {
-        Integer port = (Integer) stateConf.get(Config.TRANSACTIONAL_ZOOKEEPER_PORT);
+        Integer port = (Integer) stateConf.get(TRANSACTIONAL_ZOOKEEPER_PORT);
         String serverPorts = "";
-        for (String server : (List<String>) stateConf.get(Config.TRANSACTIONAL_ZOOKEEPER_SERVERS)) {
+        for (String server : (List<String>) stateConf.get(TRANSACTIONAL_ZOOKEEPER_SERVERS)) {
```

Comment:
I really don't understand this.  Why not use Config?  Is it because the Heron does not have a copy of org.apache.storm.Config on the classpath?  Wouldn't that be the better solutions for compatibility, especially for something core to storm like the transactional zookeeper?